akka-actor-typed-tests/src/test/scala/docs/akka/typed/RouterSpec.scala

```diff
@@ -63,15 +72,25 @@ class RouterSpec extends ScalaTestWithActorTestKit with WordSpecLike {
 
         Behaviors.empty
       })
+
+      probe.receiveMessages(10)
```

Comment:
Before it would continue with the actor running, which created a race in that the actor system could be stopped before all messages received (which is what was noticed in #27184)